loco-new/src/generator/executer/filesystem.rs

```diff
@@ -13,6 +13,12 @@ pub struct FileSystem {
     pub template_engine: generator::template::Template,
 }
 
+impl From<std::io::Error> for super::Error {
+    fn from(err: std::io::Error) -> Self {
+        super::Error::msg(err.to_string())
+    }
+}
+
```

Comment:
We use the `thiserror` crate for error handling.
But why is it necessary?

Reply:
This was needed since `std::fs::create_dir_all` returns a different Error.

But will revert that based on the comment below.